src/GitHubExtension/DeveloperId/LoginUI/EnterpriseServerPATPage.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System.Text.Json;

```

Comment:
Any reason this isn't using newtonsoft.json?

Reply:
The native library is sufficient for these needs.